assets/app/scripts/controllers/overview.js

```diff
@@ -386,7 +418,103 @@ angular.module('openshiftConsole')
       });
     });
 
+    var updateTimeout = null;
+
+    function updateTopology() {
+      updateTimeout = null;
+
+      var topologyRelations = [];
+      var topologyItems = { };
+
+      // We only add pods that are pointed to by a service
+      // Note that we use metadata.selfLink as a unique identifier since
+      // metadata.uid isn't present on everything
```

Comment:
the missing uid on the kubernetes service was fixed in https://github.com/GoogleCloudPlatform/kubernetes/pull/9845